Tests/BacktraceBreadcrumbTests.swift

```diff
@@ -163,10 +163,6 @@ final class BacktraceBreadcrumbTests: QuickSpec {
                     var writeIndex = 0
                     while writeIndex < 1000 {
                         let text = "this is Breadcrumb number \(writeIndex)"
-                        // submit a task to the queue for background execution
-                        DispatchQueue.global().async(group: group, execute: {
-                            expect { breadcrumbs.addBreadcrumb(text) }.to(beTrue())
```

Comment:
Without this background task, this test doesn't do what is intended to do.

We had lots of problems with multithreaded behavior corrupting the file, so beware!